packages/mobile/src/common/types/verify/verify.ts

```diff
@@ -2,4 +2,20 @@ type VerifyPhoneRequestDto = {
   phone: string;
 };
 
-export type { VerifyPhoneRequestDto };
+type VerifyEmailRequestDto = {
+  email: string;
+};
+
+type PropsVerifyScreens = {
+  route: {
+    params: {
+      fromSignUp?: boolean;
+    };
+  };
+};
```

Comment:
You can use navigation types for this, e.g.:
`type VerifyEmailScreenProps = NativeStackScreenProps<RootNavigationParamList, typeof RootScreenName.VERIFY_EMAIL>`

Reply:
corrected